recipes/prefsync/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.2" %}
+{% set sha256 = "66f0d3870b442e715de23b632e83f27f5971885b40bb3f6b211437138c554496" %}
+
+package:
+  name: prefsync
+  version: {{ version }}
+
+source:
+  fn: prefsync-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/prefsync/prefsync-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [not osx]
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
As you are not using `setuptools`, please just make this `python setup.py install` with no extra arguments.